packages/mdc-button/mdc-button.scss

```diff
@@ -86,14 +87,9 @@
 }
 
 .mdc-button--raised {
+  @include mdc-button-contained();
```

Comment:
Just for record keeping: I was doing this because I was worried about parameterizing them separately. But what @kfranqueiro pointed out is this `scss` file will be reflecting only baseline and theme style anyway, so it should be less of a concern within this context.